http-server/src/test/java/org/triplea/server/lobby/chat/InetExtractorTest.java

```diff
@@ -0,0 +1,21 @@
+package org.triplea.server.lobby.chat;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.net.InetAddress;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class InetExtractorTest {
+
+  private static final String IP_ADDRESS = "127.0.0.1";
+
+  @Test
+  void verify() throws Exception {
+    final Map<String, Object> propertiesMap =
+        Map.of(InetExtractor.IP_ADDRESS_KEY, "/" + IP_ADDRESS + ":42840");
```

Comment:
I'm curious to know where this address format originates from

Reply:
In INetExtractor:
```
  public static InetAddress extract(final Map<String, Object> userSession) {
    // expected format '/127.0.0.1:42840'
    final String ipString = String.valueOf(userSession.get(IP_ADDRESS_KEY)).substring(1);
```
The `userSession` comes from the `Session` object that is part of the websocket methods, ie: `session.getUserProperties`

In some initial research on how to get this value, the IP address being placed on the userSession is specific to the container, for us, 'jetty'. The format is post-priori, essentially I verified that on a running server we had this property, printed it, and hence the expected format.